app/points/calculate-community-tier-workload-points.js

```diff
@@ -0,0 +1,15 @@
+const calculatePointsForTier = require('../../app/points/calculate-points-for-tier')
+
+module.exports = function (communityTiers, caseTypeWeightings, commTierCPEnabled, commTier3AEnabled) {
+  var communityWorkloadPoints = 0
+  communityTiers.getTierCountsAsList().forEach(function (tier) {
+    communityWorkloadPoints += calculatePointsForTier(tier.tierCount, tier.points, caseTypeWeightings)
+  }, this)
+  if (commTierCPEnabled) {
+    communityWorkloadPoints += calculatePointsForTier(communityTiers.a1.tierCount, communityTiers.a1.points, caseTypeWeightings.commTierCPWeighting)
```

Comment:
According to https://docs.google.com/document/d/1kCYaMbXKaQLrfNQI-L-C7p9F8sZHqa-_Cd4gVM4jWFU/edit#heading=h.fx9njpuwzu6n the calculation for this should be `ComTier1CPTotal = ComTier1CP * WP ComTier1CP * CP Weighting` so I don't think we should be calling `calculationPointsForTier`.